data/src/main/java/com/microsoft/azure/kusto/data/ConnectionStringBuilder.java

```diff
@@ -42,6 +42,10 @@ public String getClusterUrl() {
         return clusterUri;
     }
 
+    void setClusterUrl(String clusterUri) {
```

Comment:
This isn't relevant to your change, but I just wanted to say that I suggest we make public all methods here that are part of its API (which doesn't include setClusterUrl). I think we want the other methods to be accessible even to other packages (even though Java SDK isn't currently organized).

Reply:
we should consider this but yea not for this PR